### PR TITLE
Summary Product block: Add support for global style

### PR DESCRIPTION
**Pull request by gigitux**
~This PR is blocked by #5515~

This PR adds support for global style for the `Summary Product block`.

Now, the user can edit the style for:
- text-color
- font-size

#4965 

### Screenshots

image
*without global style*

image
*with global style*

### Manual Testing

Check out this branch:

1. On `WordPress 5.9`, install and enable the Gutenberg plugin.
2. Install and enable the `Twenty Twenty-Two` theme.
3. Add the `Single Product block` (this block contains `Summary Product block`) to a post.
4. Get the focus on `Summary Product block`.
5. On the right sidebar, personalize the styles of the block.
6. Go on the page and check if there are changes.
7. Reset to default using the `Reset` button from the different sections. 
8. Go to Dashboard and select Appearance > Editor (beta).
9. On the Editor page click on the `Styles` icon on the right-top corner.
10. Verify that the `Summary Product block` is shown under the `Blocks` section. Personalize again the block.
11. Save your changes.
12. Go on the page created earlier and check if all styles are applied correctly.
13. Edit your previous post/page again.
14. Change again the styles.
15. Save your changes.
16. Check if these styles have priority over the styles from the Site Editor.

### Changelog

> Add support for the global style for the Summary Product block.

### CODE REVIEW

File: assets/js/atomic/blocks/product-elements/summary/block.js [hunk 1]
```diff
@@ -23,9 +28,13 @@ import './style.scss';
...
  * @return {*} The component.
  */
-const Block = ( { className } ) => {
+const Block = ( props ) => {
+	const { className } = props;
+
 	const { parentClassName } = useInnerBlockLayoutContext();
 	const { product } = useProductDataContext();
+	const colorProps = useColorProps( props );
+	const typograhyProps = useTypographyProps( props );
```
Comment (sunyatasattva) [inline assets/js/atomic/blocks/product-elements/summary/block.js]: Typo in `typography`

**Review by sunyatasattva — CHANGES_REQUESTED** [review]
Just a small typo, everything else LGTM

File: assets/js/atomic/blocks/product-elements/summary/block.js (hunk 1 above)
Comment (gigitux) [reply assets/js/atomic/blocks/product-elements/summary/block.js]: Thanks!

**Review by sunyatasattva — APPROVED** [review]
LGTM!

**Pull request merged**